internal/journal/voljournal.go

```diff
@@ -182,6 +186,7 @@ func NewCSISnapshotJournal(suffix string) *Config {
 		namespace:               "",
 		csiImageIDKey:           "csi.imageid",
 		encryptKMSKey:           "csi.volume.encryptKMS",
+		ownerKey:                "csi.volume.owner",
```

Comment:
is this needed for a snapshot also?

Reply:
I'd say it is needed where the `csi.volume.encryptKMS` is used, otherwise the options to would not be complete.